library/lib/build.js

```diff
@@ -291,7 +291,7 @@ module.exports = function build({ watch }) {
 
   function gatherEntries() {
     return walkSync(srcDir, { globs: ['**/*.*.js', '**/*.entry.css'] }).reduce(
-      (result, entry) => (result[entry] = './' + entry, result),
+      (result, entry) => ({ ...result, [entry]: './' + entry }),
```

Comment:
Is this the result of a rule?

Reply:
Yes, the [https://eslint.org/docs/rules/no-return-assign(`no-return-assign`)-rule.
This would also pass:
```js
(result, entry) => ((result[entry] = './' + entry), result),

Reply:
Well, it seems the rule is wrong, the statement is not in return position. We should file a bug.

We can not use `...` for objects as it is not in the version of node that runs on our servers.

Reply:
https://github.com/eslint/eslint/issues/9418